db/sync.go

```diff
@@ -11,6 +11,8 @@ import (
 )
 
 type SyncInfo struct {
+	// SyncID uniquely identifies this sync for the purposes of logging and debugging.
+	SyncID string
```

Comment:
[fix ]Do we want this in the bindings?

If yes, fix the name.
If no, `json:"-,"`

Reply:
yowch, fixing now, thanks for the catch. yes, i would like you to pass it back into maybeendsync.